### DISCUSSION TITLE
请教下该库有无包含ColumnLine组件，该如何实现？

### DISCUSSION THREAD

**Discussion by kfcvs** [open-data-plan/g2plot-vue#773] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/31976998/166924676-a2812972-af1f-45d6-83ae-1602bd3369ec.png)

**Comment by kagawagao** [comment, marked as answer]
[![Edit g2plot-vue-basic-example (forked)](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/g2plot-vue-basic-example-forked-hrs7kr?fontsize=14&hidenavigation=1&theme=dark)

**Reply by kfcvs** [reply]
收到，非常感谢!
